gradle/solr/force-versions.gradle

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+// Force slf4j version so that it does not get auto promoted from 1x to 2-alpha due to Jetty dependency.
+
+project(":solr").subprojects {
+  plugins.withType(JavaPlugin) {
+    dependencies {
+      implementation enforcedPlatform('org.slf4j:slf4j-api:1.7.35')
```

Comment:
Should we set this version in version.props like the rest of our libraries?

Reply:
I think this was done this way because you cannot easily do that here, or I assume Dawid would have, but I'll see what happens.

Reply:
Think it actually works.